en/docs/concepts/user-management/users.md

```diff
@@ -0,0 +1,58 @@
+# Users
+
+A user is the digital representation of a physical user who interacts with applications. A user can be a digital identity of a human or a machine. A user can be recognized by a number of attributes. These are called **user claims**. User claims are data that defines the user, i.e., anything the user is, owned by, or associated with.
+
+A unique **User account** is created for each user. Then they are assigned roles with a certain set of permissions. We will further discuss roles and permissions in the [next section](../../concepts/user-management/roles-and-permissions.md). 
```

Comment:
Let's try and avoid using "you" and "we" in official docs wherever we can. In this case, we can rephrase it to something like "Roles and permissions are explained further in the next section"